src/components/Tooltip/Tooltip.tsx

```diff
@@ -104,6 +107,18 @@ export const Tooltip: FC<TooltipProps> = React.memo(
         `${tooltipId?.current}-reference`
       );
 
+      const isMobile: boolean = isIOS() || isAndroid();
```

Comment:
might be a reasonable exported method like isTouching. most scenarios will want the generic over the specific os I would think

Reply:
When it comes to screen readers we'll need the specificity.